scripts/docker-compose/imports/archiefbeheercomponent-database/database/1-setup-applicatie.sql

```diff
@@ -0,0 +1,2 @@
+-- create superuser to be able to log in to the UI with username admin and password admin
+INSERT INTO accounts_user (id, password, last_login, is_superuser, username, first_name, last_name, email, is_staff, is_active, date_joined) VALUES (1, 'pbkdf2_sha256$260000$gtIe19cI1vW9RzIsRDpriC$o8G6cItI5vXqbGFcXuu0pbullajpvMDc6Hze70mf+jE=', null, true, 'admin', '', '', 'admin@example.com', true, true, '2023-08-08 15:14:56.735552 +00:00');
```

Comment:
Is there not an environment variable for this? Maykin added all sorts of env vars to their components recently. Definitely also for adding admin accounts.

Reply:
Documentation does not mention an environment variable for superuser. I could not find something around it in the code either.

Instead there's a python script to load roles, users and emails that requires ABC to be up&running, which means it's not much different than what we currently have with our scripts :(